apps/wallet/src/ui/app/components/accounts/AccountItem.tsx

```diff
@@ -31,6 +35,10 @@ export const AccountItem = forwardRef<HTMLDivElement, AccountItemProps>(
 		const copyAddress = useCopyToClipboard(account?.address!, {
 			copySuccessMessage: 'Address copied',
 		});
+		const explorerHref = useExplorerLink({
+			type: ExplorerLinkType.address,
+			address: account?.address!,
```

Comment:
I don't think address/account is always available here, `!` is the best thing to use here. Probably we need to update the `useExplorerLink` to accept undefined/null